### ISSUE TITLE
Why val_loss is always 0?

### ISSUE THREAD

**Issue by gxwangupc** [cyvius96/adgpm#5]
hi,
I get val_loss=0.0000 when run "python train_gcn_*.py". Why is it ?
Thank you.

**Comment by yinboc** [comment]
Thanks for your question.

Because by default the train/val split is 10/0 (see the code), that is, no validation set of nodes.
This is mainly because every node is important for having a good performance when testing its neighbors.

**Comment by gxwangupc** [comment]
Thank you for your reply.
